README.md

```diff
@@ -52,20 +52,14 @@ The `make install` command may not require `sudo` depending on your choice of in
 ## Documentation
 
 There is extensive online documentation.
-You'll find all the details at:
-
-    https://taskwarrior.org/docs/timewarrior
-
-At the site you'll find online documentation, downloads, news and more.
+You'll find all the details at [taskwarrior.org](https://taskwarrior.org/docs/timewarrior).
```

Comment:
Because this links goes to taskwarrior.org and not timewarrior.net, I would prefer to see the whole URL (taskwarrior.org/docs/timewarrior) in the link text.